### PR TITLE
update translated UI texts

### PR DESCRIPTION
**Pull request by DRracer**

### CODE REVIEW

**Review by XPila — APPROVED** [review]
pouze v config.h/52 je spatne velikost v bytes (ma byt 12288)

File: lang/lang_en.txt [hunk 1]
```diff
@@ -930,3 +930,147 @@
...
+"Select"
+
+#
+"Sensor info"
+
+#
+"Sheet"
+
+#
+"Sound     [assist]"
```
Comment (3d-gussner) [inline lang/lang_en.txt:1064]: After following https://github.com/DRracer/Prusa-Firmware/blob/translations-doc/lang/translations.md the `not_tran.txt` showed that `Sound    [assist]` is missing.
Checking the `lang_en.txt` and `lang_en_??.txt` it seams to be a typo using one space too much.

The message should be 17 chars long and not 18 with 4 spaces between `Sound` and `[assist]`

File: lang/lang_en_cz.txt [hunk 2]
```diff
@@ -1240,4 +1240,196 @@
...
+#
+"Sensor info"
+"Senzor info"
+
+#
+"Sheet"
+"Plat"
+
+#
+"Sound     [assist]"
```
Comment (3d-gussner) [inline lang/lang_en_cz.txt:1418]: After following https://github.com/DRracer/Prusa-Firmware/blob/translations-doc/lang/translations.md the `not_tran.txt` showed that `Sound    [assist]` is missing.
Checking the `lang_en.txt` and `lang_en_??.txt` it seams to be a typo using one space too much.

The message should be 17 chars long and not 18 with 4 spaces between `Sound` and `[assist]`

File: lang/lang_en_cz.txt [hunk 3]
```diff
@@ -1240,4 +1240,196 @@
...
+"Zvuk      [Asist.]"
```
Comment (3d-gussner) [inline lang/lang_en_cz.txt:1419]: Must be `Zvuk     [Asist.]`

File: lang/lang_en_de.txt [hunk 4]
```diff
@@ -1241,4 +1241,196 @@
...
+#
+"Sensor info"
+"Sensor Info"
+
+#
+"Sheet"
+"Blech"
+
+#
+"Sound     [assist]"
```
Comment (3d-gussner) [inline lang/lang_en_de.txt:1419]: After following https://github.com/DRracer/Prusa-Firmware/blob/translations-doc/lang/translations.md the not_tran.txt showed that Sound [assist] is missing.
Checking the lang_en.txt and lang_en_??.txt it seams to be a typo using one space too much.

The message should be 17 chars long and not 18 with 4 spaces between Sound and [assist]

File: lang/lang_en_de.txt [hunk 5]
```diff
@@ -1241,4 +1241,196 @@
...
 "\x00"
```
Comment (3d-gussner) [inline lang/lang_en_de.txt:1420]: Should be `Sound    [Assist]`

File: lang/lang_en_es.txt [hunk 6]
```diff
@@ -1236,8 +1236,200 @@
...
+#
+"Sensor info"
+"Info sensor"
+
+#
+"Sheet"
+"Lamina"
+
+#
+"Sound     [assist]"
```
Comment (3d-gussner) [inline lang/lang_en_es.txt:1418]: After following https://github.com/DRracer/Prusa-Firmware/blob/translations-doc/lang/translations.md the not_tran.txt showed that Sound [assist] is missing.
Checking the lang_en.txt and lang_en_??.txt it seams to be a typo using one space too much.

The message should be 17 chars long and not 18 with 4 spaces between Sound and [assist]

File: lang/lang_en_es.txt [hunk 7]
```diff
@@ -1236,8 +1236,200 @@
...
+"Sonido  [asistido]"
```
Comment (3d-gussner) [inline lang/lang_en_es.txt:1419]: Should be `Sonido [asistido]`

File: lang/lang_en_fr.txt [hunk 8]
```diff
@@ -1240,4 +1240,196 @@
...
+#
+"Sensor info"
+"Info capteur"
+
+#
+"Sheet"
+"Feuille"
+
+#
+"Sound     [assist]"
```
Comment (3d-gussner) [inline lang/lang_en_fr.txt:1418]: After following https://github.com/DRracer/Prusa-Firmware/blob/translations-doc/lang/translations.md the not_tran.txt showed that Sound [assist] is missing.
Checking the lang_en.txt and lang_en_??.txt it seams to be a typo using one space too much.

The message should be 17 chars long and not 18 with 4 spaces between Sound and [assist]

File: lang/lang_en_fr.txt [hunk 9]
```diff
@@ -1240,4 +1240,196 @@
...
+"Son       [Assist]"
```
Comment (3d-gussner) [inline lang/lang_en_fr.txt:1419]: Should be `Son      [Assist]`

File: lang/lang_en_it.txt [hunk 10]
```diff
@@ -1241,3 +1241,195 @@
...
+#
+"Sensor info"
+"Info Sensore"
+
+#
+"Sheet"
+"Piano"
+
+#
+"Sound     [assist]"
```
Comment (3d-gussner) [inline lang/lang_en_it.txt:1418]: After following https://github.com/DRracer/Prusa-Firmware/blob/translations-doc/lang/translations.md the not_tran.txt showed that Sound [assist] is missing.
Checking the lang_en.txt and lang_en_??.txt it seams to be a typo using one space too much.

The message should be 17 chars long and not 18 with 4 spaces between Sound and [assist]

File: lang/lang_en_it.txt [hunk 11]
```diff
@@ -1241,3 +1241,195 @@
...
+"Suono  [assistito]"
```
Comment (3d-gussner) [inline lang/lang_en_it.txt:1419]: Should be `Suono [assistito]`

File: lang/lang_en_pl.txt [hunk 12]
```diff
@@ -1236,8 +1236,200 @@
...
+#
+"Sensor info"
+"Info o sensorach"
+
+#
+"Sheet"
+"Plyta"
+
+#
+"Sound     [assist]"
```
Comment (3d-gussner) [inline lang/lang_en_pl.txt:1418]: After following https://github.com/DRracer/Prusa-Firmware/blob/translations-doc/lang/translations.md the not_tran.txt showed that Sound [assist] is missing.
Checking the lang_en.txt and lang_en_??.txt it seams to be a typo using one space too much.

The message should be 17 chars long and not 18 with 4 spaces between Sound and [assist]

File: lang/lang_en_pl.txt [hunk 13]
```diff
@@ -1236,8 +1236,200 @@
...
+"Dzwiek    [asyst.]"
```
Comment (3d-gussner) [inline lang/lang_en_pl.txt:1419]: Should be `Dzwiek   [asyst.]`

**Review by 3d-gussner — COMMENTED** [review]
After following https://github.com/DRracer/Prusa-Firmware/blob/translations-doc/lang/translations.md the not_tran.txt showed that Sound [assist] is missing.
Checking the lang_en.txt and lang_en_??.txt it seams to be a typo using one space too much.

The message should be 17 chars long and not 18 with 4 spaces between Sound and [assist]

**Comment by leptun** [comment]
@3d-gussner Good catch. This is one of the situations where the toggles implementation would prevent this kind of issues from occurring. I hope that next week I can finish it with DRracer and have it ready for the first upcomming 3.9 beta. First thing to do is integrate the new translations from this PR in my PR. Hopefully it’s easy enough.

**Pull request merged**